### PR TITLE
SNET-1229 Added registration id to display

### PR DESCRIPTION
**Pull request by dileep2503**

### CODE REVIEW

File: components/Registration/index.tsx [hunk 1]
```diff
@@ -436,7 +437,10 @@ const Registration: FunctionComponent<RegistrationProps> = ({
         airdrop_id: activeWindow?.airdrop_id,
         airdrop_window_id: activeWindow?.airdrop_window_id,
       };
-      await axios.post('airdrop/registration', payload);
+      await axios.post('airdrop/registration', payload).then((response) => {
```
Comment (raamb) [inline components/Registration/index.tsx:440]: Remove console.log

File: components/Registration/index.tsx [hunk 2]
```diff
@@ -568,3 +573,6 @@ const Registration: FunctionComponent<RegistrationProps> = ({
 };
 
 export default Registration;
```
Comment (raamb) [inline components/Registration/index.tsx:575]: What is this?

File: components/Registration/index.tsx [hunk 3]
```diff
@@ -65,6 +65,7 @@ const Registration: FunctionComponent<RegistrationProps> = ({
 }) => {
```
Comment (raamb) [inline components/Registration/index.tsx:65]: Please add a meaningful description to the PR. Also reference the issue number

File: components/Registration/index.tsx (hunk 1 above)
Comment (dileep2503) [reply components/Registration/index.tsx:440]: Removed console.log

File: components/Registration/index.tsx (hunk 2 above)
Comment (dileep2503) [reply components/Registration/index.tsx]: Resolved sir

File: components/Registration/index.tsx (hunk 3 above)
Comment (dileep2503) [reply components/Registration/index.tsx:65]: Ok sir

**Review by raamb — APPROVED** [review]

**Pull request merged**